tests/test_statistics.py

```diff
@@ -0,0 +1,46 @@
+from pathlib import Path
+
+import pytest
+
+from modules.statistics import Statistics
+
+
+@pytest.fixture
```

Comment:
Maybe consider to leave a code comment on the purpose of the decorator.
I have at least never seen this decorator before, so a description would be nice

Reply:
Good question!

If you annotate a function with `@pytest.fixture` you make the result of that function available as an input for other test functions. Here is an example:

```{python}
@pytest.fixture
def my_number():
    return 2

def test_my_number(my_number):
    assert my_number == 2
```

That way you reduce the boilerplate in your test code if you need to create the same object over and over again in all your tests.

Does this make sense? If so, I think a comment is unecessary, as the decorator will appear a lot of places in a code base that is tested, and you never know which decorator a new contributor sees first.